service/src/main/java/com/epam/esm/config/ServiceConfig.java

```diff
@@ -0,0 +1,9 @@
+package com.epam.esm.config;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan("com.epam.esm")
+public class ServiceConfig {
+}
```

Comment:
Are you sure, that you need this empty configuration?

Reply:
No